Blogpost1.php

```diff
@@ -0,0 +1,28 @@
+<html>
+	<head>
+		<title>May 17th blog</title>
+
+		<link rel='stylesheet' href="blogpost.css">
+
+		<link href='https://fonts.googleapis.com/css?family=Barrio' rel='stylesheet'>
+
+		<link href='https://fonts.googleapis.com/css?family=Delius Unicase' rel='stylesheet'>
+
+	</head>
+	<body class="backdrop">
+		<h1> My Interests</h1>
+		<img class="test" src="/beyhive.jpg" alt="Beyhive">
```

Comment:
This is fine, but as your code base gets bigger, you'll want to organize your files a bit. Maybe make a folder called `/images` where you can put all the images so they don't get mixed up with your html/php files